cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -363,20 +363,21 @@ def _check_in_out_access(security_groups_ids, port):
         # Check all inbound rules
         for rule in sec_group.get("IpPermissions"):
             if _check_sg_rules_for_port(rule, port):
-                in_access = True
-                break
+                if is_cidr_optional or rule.get("IpRanges") or rule.get("PrefixListIds"):
```

Comment:
It seems to me that when `is_cidr_optional` is set to True we're skipping everything and returning always True, so I think we don't really need this parameter.

When all security groups are customized we can simply avoid to call the `_check_in_out_access` instead of calling it with `is_cidr_optional` set to True, see comment below.

Reply:
See comment below